www/posts/2020-05-25-v2-release-day.md

```diff
@@ -0,0 +1,152 @@
+
+<blockquote class="twitter-tweet" data-conversation="none" data-dnt="true"><p lang="en" dir="ltr">Check out the network tab to see unbundled development in action:<br><br>1️⃣ App.js is changed<br>2️⃣ App.js is refetched via HMR<br>3️⃣ App.js is rebuilt from source<br>✅ Total time: a few milliseconds. <br><br>Zero bundling, rebundling, or multi-file compiling needed for the snappiest update possible</p>&mdash; Pika 📦 (@pikapkg) <a href="https://twitter.com/pikapkg/status/1264245654970232832?ref_src=twsrc%5Etfw">May 23, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+**Bundling during development slows you down.**  It may be hard to see, since most of us haven't done meaningful web development *without* a bundler in ages. But under the hood, bundled dev servers like `webpack-dev-server` & Create React App need to load, build, resolve and bundle your application before you can serve a single file. Smart caching can speed this up somewhat, but it's still common for a dev servers to take 30+ seconds to startup. And that's not even counting the time you spend re-bundling your application after you make a change, which can take another several seconds before you see those changes reflected in the browser.
+
+**You can think of bundling as a step with `O(n)` complexity:** As the size of your project grows, your bundler needs to do more work to startup your app. Depending on how your bundler is designed, it may even have `O(n^2)` complexity: as your project grows, your dev environment gets ***exponentially*** slower as it handles code splitting and every possible bundle permutation. Neither scenario is ideal when projects can grow to hundreds or even thousands of files. 
+
+## The Future: O(1) Build Tooling
+
+![webpack vs. snowpack diagram](/img/snowpack-unbundled-example-3.png)
+
+Snowpack is a move away from complex bundled dev environments and towards faster, unbundled development. Unbundled development introduces a new, lighter complexity story for web development where you can move faster by simply doing less unnecessary work.
+
+**Snowpack is a platform for O(1) build tooling.**  This is a term I first heard used by CodeSandbox's [Ives van Hoorne](https://www.youtube.com/watch?v=Yu9zcJJ4Uz0) that I think perfectly captures what makes Snowpack's workflow so great: when a file changes, only that file is rebuilt. There's no larger rebundling or application rebuilding needed, just a simple, linear, deterministic `input -> build -> output` workflow.
+
+- O(1) build tooling is easier to reason about, since your build input is always a single file.
```

Comment:
`since`, `because`, `since`, I would remove all of these and the details following to make points more concise